draft-ietf-quic-ack-frequency.md

```diff
@@ -411,18 +411,25 @@ controllers.
 ## Congestion Control
 
 A sender needs to be responsive to notifications of congestion, such as
-a packet loss or an ECN CE marking. Also, window-based congestion controllers
-that strictly adhere to packet conservation, such as the one defined in
-{{QUIC-RECOVERY}}, rely on receipt of acknowledgments to send additional data into
-the network, and will suffer degraded performance if acknowledgments are delayed
-excessively.
-
-To enable a sender to respond to potential network congestion, a sender SHOULD
-cause a receiver to send an acknowledgement at least once per RTT if there are
-unacknowledged ack-eliciting packets in flight. A sender can accomplish this by
-sending an IMMEDIATE_ACK frame once per round-trip time (RTT), or it can set the
-Ack-Eliciting Threshold and Request Max Ack Delay values to be no more than a
-congestion window and an estimated RTT, respectively.
+a packet loss or an ECN CE marking.  To enable a sender to respond to potential
+network congestion, a sender SHOULD cause a receiver to send an acknowledgement
+at least once per RTT if there are unacknowledged ack-eliciting packets in flight.
+A sender can accomplish this by sending an IMMEDIATE_ACK frame once per
+round-trip time (RTT), or it can set the Ack-Eliciting Threshold and
+Request Max Ack Delay values to be no more than a congestion window and an
+estimated RTT, respectively.
+
+A congestion controller that is congestion window limited relies upon receiving
+acknowledgements to send additional data into the network.  An increase in
+acknowledgement delay increases the delay in sending data, which can reduce the
+achieved bandwidth.  Congestion window growth can also depend upon receiving
+acknowledgements, such as in slow start (Section 7.3.1 of {{QUIC-RECOVERY}}),
+so delaying acknowledgements can delay the increase in congestion window.
+
+Window-based congestion controllers, such as the one defined in {{QUIC-RECOVERY}},
+rely on receipt of acknowledgments to send additional data into the network,
+so performance is degraded if acknowledgments are delayed excessively.
```

Comment:
This looks like a repetition of the more precise text in the previous paragraph.

Reply:
Agreed, it's the leftover text from what I started with.  Should I just delete it?

Reply:
To me though for Internet paths, the key issue is not the "performance" reduction, the primary concern of the transport area was the increased congestion risk - especially to other flows that share a common bottleneck along the path.

Reply:
What increased congestion are you concerned about by excessively delayed acks?

Reply:
What I mean is a sluggish response to congestion when the sender was neither cwnd nor flow credit-limited (i.e. sender can still send without receiving feedback) and the ACK frequency is low (>>IW packets per ACK), where the sender persists sending new data into the network after the congestion event, i.e. CE-mark or loss.

Reply:
This paragraph is talking about the congestion controller potentially being CWND limited and not being able to send.  It's not discussing anything related to congestion (ie: CE or missing packets).

Reply:
As long as the ID has the CC issues as well as the performance ones, I'll be happy.

Reply:
I think risks about not receiving timely feedback are covered in the initial paragraph of this PR. Let's merge this for now and see if anything is still needed in addition.